File: ch1/c/code1_3.c
#include <stdio.h>

int sum(int n)
{
	return (n <= 0)? 0: ((n == 1)? 1: (n + sum(n - 1)));
}

int main()
{  
  printf("숫자를 하나 입력하세요 :");
  int x;
  scanf("%d", &x);
  printf("1에서 %d 까지의 합 : %d\n", x, sum(x));
  return 0;
}

File: ch1/python/code1_3.py
def sum(n):
  return 0 if n <= 0 else (1 if n == 1 else n + sum(n - 1))

n = int(input('숫자를 하나 입력하세요 : '))
print('1에서 %d 까지의 합 : %d' % (n, sum(n)))
  